sentry/src/main/resources/META-INF/native-image/io.sentry/sentry/reflect-config.json

```diff
@@ -0,0 +1,146 @@
+[
```

Comment:
missing the envelope classes that rely on serialization too eg SentryEnvelopeItem ?

Reply:
Hm I'll integration works without these classes. Just in case I'll add them in case I missed some use case.